src/constants.js

```diff
@@ -36,3 +36,5 @@ export const DTYPE_VALUES = {
     TypedArray: Float32Array
   }
 };
+
+export const NO_WEBGL2 = !document.createElement('canvas').getContext('webgl2');
```

Comment:
A little worried about having a global like this. Hard to think of a side effect off the top of my head, but it might be nicer to have a `useWebGL2` prop for the layers, and then do feature detection in the viewers and pass this as a prop.